### PR TITLE
1.0

### PR DESCRIPTION
**Pull request by BeryJu**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # authentik
 
-![Version: 1.0.0-RC7](https://img.shields.io/badge/Version-1.0.0--RC7-informational?style=flat-square) ![AppVersion: 2021.5.2](https://img.shields.io/badge/AppVersion-2021.5.2-informational?style=flat-square)
+![Version: 1.0.0](https://img.shields.io/badge/Version-1.0.0--RC7-informational?style=flat-square) ![AppVersion: 2021.5.2](https://img.shields.io/badge/AppVersion-2021.5.2-informational?style=flat-square)
```
Comment (dirtycajunrice) [inline README.md:3]: I assume this isnt going to merge until 2021.5.3 when the flow ui is fixed?

**Review by dirtycajunrice — CHANGES_REQUESTED** [review]

File: charts/authentik/templates/deployment.yaml [hunk 2]
```diff
@@ -5,6 +5,7 @@ kind: Deployment
 metadata:
   name: {{ printf "%s-%s" (include "common.names.fullname" $) . }}
   labels:
+    app.kubernetes.io/component: {{ . }}
   {{- include "common.labels" $ | nindent 4 }}
```
[the remark this answers is not in the document]
Comment (BeryJu) [reply charts/authentik/templates/deployment.yaml:9]: @dirtycajunrice pretty much all of these changes are in relation to https://github.com/goauthentik/helm/issues/7 , I don't need the component label for anything application side, but it makes sense that it is set

Also according to helm template and the CI tests, the indentation is correct as is, or is your suggestion just for cleanliness?

File: README.md (hunk 1 above)
Comment (BeryJu) [reply README.md:3]: Yeah I'm waiting with 2021.5.3 until I have checked all the forms to make sure I'm not missing anything

File: charts/authentik/templates/deployment.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dirtycajunrice) [reply charts/authentik/templates/deployment.yaml]: cleanliness

[the remark this answers is not in the document]
Comment (BeryJu) [reply charts/authentik/templates/deployment.yaml]: right, fixed that in the latest commit

**Pull request merged**